controllers/addon_controller.go

```diff
@@ -86,10 +91,19 @@ type AddonReconciler struct {
 	generatedClient *kubernetes.Clientset
```

Comment:
This can probably be changed to be `kubernetes.Interface`

Reply:
c95e1124592857e40df10eaa69c41ffcbe6b6a95